exporter.go

```diff
@@ -63,75 +66,133 @@ func hashNameAndLabels(name string, labels prometheus.Labels) uint64 {
 	return hash.Sum64()
 }
 
+func labelsNames(labels prometheus.Labels) []string {
+	names := make([]string, 0, len(labels))
+	for labelName := range labels {
+		names = append(names, labelName)
+	}
+	sort.Strings(names)
+	return names
+}
+
+// hashNameAndLabels returns a hash value of the provided name string and all
```

Comment:
the name here does not match the function definition

Reply:
I see that you left `hashNameAndLabels` in – is it still used? why?

Reply:
Will clean it up!

Reply:
`hashNameAndLabels` is used to calculate a key for label values in Exporter.labelValues.